src/figuresClasses.ts

```diff
@@ -1,11 +1,119 @@
-export interface Figure {}
+export enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
+
+export enum Color {
+  Red = 'red',
+  Green = 'green',
+  Blue = 'blue',
+}
+
+function validatePositiveNumbers(...numbers: number[]): void {
+  if (numbers.some((n) => n <= 0)) {
+    throw new Error('Values must be greater than 0');
+  }
+}
+
+function roundToTwo(num: number): number {
+  return Math.floor(num * 100) / 100;
+}
 
-export class Triangle implements Figure {}
+export interface Figure {
+  shape: Shape;
+  color: string;
```

Comment:
The `color` property in the `Figure` interface should be of type `Color` instead of `string` to ensure type safety and consistency with the `Color` enum.